### ISSUE TITLE
Странности с обменом валюты

### ISSUE THREAD

**Issue by FedorovB** [shcherbakov-vadim/1c-money-custom-issues#38]
Вадим, добрый день.
На днях совершил покупку евро, что, разумеется, отразил в базе. Но эта операция формирует какие-то странные данные в отчете по расходам. Вот сама операция:
![image](https://github.com/user-attachments/assets/e1c85348-8fc7-4e5e-a29a-8d07cf7f35d4)
А в отчете "Расходы по статьям" данная операция выглядит очень странным образом. Она идет с пустой статьей расходов и формирует сумму, как разница между проданной суммой и купленной. То есть, я отдал 111000 руб., получил 1000 евро, а в отчете это выглядит как 111000 - 1000 = 110000. По моему, тут ошибка.. :(
![image](https://github.com/user-attachments/assets/bb0e950e-ca52-4b85-a8e2-d764fd5ca6ed)
Посмотрите пожалуйста. Заранее Вам благодарен.

**Comment by FedorovB** [comment]
При чем, эта операция формирует этот непонятный расход почти во всех отчетах. Увидел в динамике доходов и расходов, а так же в диаграмме на главной странице:
![image](https://github.com/user-attachments/assets/7ab54493-16fc-4234-9963-a0d83af4ea9b)

**Comment by shcherbakov-vadim** [comment]
Вы можете открыть через "Функции для технического специалиста" регистр бухгалтерии "Журнал операций" и посмотреть какие движения сформировал ваш документ "Обмен валюты"

**Comment by FedorovB** [comment]
Вот, вкладка "Доход":
![image](https://github.com/user-attachments/assets/a74e187d-b4f6-474d-8ed1-6505d0fb32df)
И "Расход":
![image](https://github.com/user-attachments/assets/69abc834-dea7-470c-bd09-c9bc09a70984)

**Comment by shcherbakov-vadim** [comment]
Нет, нужно открыть в главном меню "Функции технического специалиста..." и там открыть регистр бухгалтерии "Журнал операций" и посмотреть какие движения сформировал ваш документ "Обмен валюты"
По движениям, как будто у вас до этого уже были продано в минус 1000 евро по 1 рублю

**Comment by FedorovB** [comment]
1)
![image](https://github.com/user-attachments/assets/840dd8e0-08f0-4d45-a154-c676ae79832c)
2) (перемотал вправо)
![image](https://github.com/user-attachments/assets/9d7eb81b-4a01-4212-b184-7ca9f5e852a6)

**Comment by shcherbakov-vadim** [comment]
Удалось воспроизвести проблему. Она возникает если в настройках снят флаг "Формировать финансовый результат при продаже валюты". Будет исправлено в следующей версии кастомизированной конфигурации

**Comment by shcherbakov-vadim** [comment]
https://chtch.ru/ru/1sdengi/#2.0.44.28.46
